3cn-ecn/nantralPlatform
# Home page and event update

# Description

- Created a button to display bde's planning (this is a test version and is not linked with backend rn)
- Refactor of home page
  - Replaced "My events" section with "This week" Section
  - Created eventSection component


# Tests

1. Steps to reproduce :

- Make sure you have added events to the database.
- Go to the home page.
- Enjoy !

# Screenshots
![image](https://user-images.githubusercontent.com/112650656/219688371-53e99d33-0c6c-4ddb-853a-341335b284b4.png)
![image](https://user-images.githubusercontent.com/112650656/219688471-1d79ad3d-b472-4d37-93ab-50d0c3403f75.png)
![image](https://user-images.githubusercontent.com/112650656/219688557-d5e87b4c-8742-4f9c-9d2e-03f57588d114.png)
Desktop version
![image](https://user-images.githubusercontent.com/112650656/219714021-b05c761b-b346-467a-b2b7-acd509f3a309.png)
Mobile version
![image](https://user-images.githubusercontent.com/112650656/219714264-a1a55034-5a97-4e0c-940b-9518ec3a609f.png)
